### PR TITLE
Update Microsoft.Azure.Cosmos.json to 3.35.4 release

### PR DESCRIPTION
**Pull request by ealsur**
SDK 3.35.4 was released and API docs have not been updated for a while.

Closes https://github.com/Azure/azure-cosmos-dotnet-v3/issues/4084

### CODE REVIEW

**Comment by ealsur** [comment]
@danieljurek / @JimSuplizio any of you folks can help review this PR?

**Review by danieljurek — APPROVED** [review]
LGTM. @ealsur, have there been any changes to namespaces?

**Comment by ealsur** [comment]
@danieljurek No, no namespaces changed in the versions

**Comment by ealsur** [comment]
Let me know what steps are required after approval to merge this PR (I personally cannot do it)

**Review by philipthomas-MSFT — APPROVED** [review]

**Pull request merged**
